lib/dfe/analytics.rb

```diff
@@ -103,6 +104,9 @@ def self.initialize!
           model.include(DfE::Analytics::Entities)
         end
       end
+
+      DfE::Analytics::Initialise.trigger_initialise_event
```

Comment:
We already have an init routine in `railtie.rb` — could we use that?

Reply:
In `railtie.rb`, I find the App isn't fully initialised, although it depends on where in railtie the methods are called. I suppose it could be called in the `after_initialize` hook  after the `DfE::Analytics.initialize!`call, after which the config is fully initialised, as we need the config loaded before calling `trigger_initialise_event`.

I'll give that a try.